src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/DebugPropertyPage.xaml

```diff
@@ -15,7 +15,7 @@
 
   <StringProperty Name="DebugPagePlaceholderDescription"
                   DisplayName="Ignored"
-                  Description="The management of launch profiles has moved to a dedicated dialog. It may be accessed via the link below.">
+                  Description="The management of launch profiles has moved to a dedicated dialog. It may be accessed via the link below, via the Debug menu in the menu bar, or via the Debug Target command on the Standard tool bar.">
```

Comment:
nit: Is "toolbar" a single word at this point in the evolution of software? 🤓

Reply:
Not in WinForms/WPF, where it's `ToolBar` rather than `Toolbar`.

Reply:
I didn't mean context-specific to the UI engine used. The user doesn't know/care about that (they don't see the names of the controls). I meant as a concept in software, such as:
- https://en.wikipedia.org/wiki/Toolbar
- https://www.merriam-webster.com/dictionary/toolbar
- https://docs.microsoft.com/en-us/windows/apps/design/controls/menus-and-context-menus (subsection called "Menus and toolbars")
- https://docs.microsoft.com/en-us/windows/win32/uxguide/cmd-toolbars (old guide)